Code/Framework/AzCore/AzCore/std/algorithm.h

```diff
@@ -46,13 +46,18 @@ namespace AZStd
     template<class T>
     constexpr pair<T, T> minmax AZ_PREVENT_MACRO_SUBSTITUTION (const T& left, const T& right) { return AZStd::minmax(left, right, AZStd::less<T>()); }
 
+    // Returns an iterator to the largest element in the range [first, last]
+    using std::max_element;
+    // Returns an iterator to the smallest element in the range [first, last]
+    using std::min_element;
```

Comment:
update: Please use the form `[first, last)` (and change the comment below back) - this is a convention to mean 'half-open' range, where `last` is not included in the range. See https://stackoverflow.com/a/13067115 for more details